### ISSUE TITLE
[Transport] Add option cleanup

### ISSUE THREAD

**Issue by sybrew** [sybrew/the-seo-framework#621]
For [Transport](https://theseoframework.com/extensions/transport/), add a checkbox to clean the options of an SEO plugin. 

Cleaning the options should also deactivate the SEO plugin (and related plugins, like Pro versions or add-ons).
This is necessary to prevent automatic repopulation of the options and further annoyances, such as the setup wizard.

**Comment by sybrew** [comment]
From [Vladimir's comment](https://wordpress.org/support/topic/the-seo-framework-transport-%ce%b2eta-not-working/#post-16708824):

> > Sybre wrote: TSF cleans up all transported data of Rank Math, so your database is squeaky clean right after transporting.
> 
> it’s in vain! It turns out that if a person just wants to see how your plugin works, try it, then you erase the database for him!
> 
> This is sabotage!
>
> if a person has not made a backup copy before the transfer, then he will lose part of the site data!

I agree. We should not rewrite or automatically clean up the data but only copy every data point instead. 
Then, only delete the old data if the user so chooses via a checkbox.

This might be tricky when the meta keys match -- but this is only an issue with Genesis (from which we copied the keys).

An edge case we might face is when there's no more disk space. However, this is rare and solvable by the user manually -- on failure, the logger should report there's no writable disk space, though I might want to test that and add it to the FAQ (just in case).

**Comment by kekkster** [comment]
Hi @sybrew,

We’re planning to migrate several sites from Rank Math to The SEO Framework, but we’re concerned that Transport still deletes the original Rank Math metadata during migration.

Is optional cleanup or a copy-only mode still planned? Also, do you consider the current released version of Transport fully compatible with recent Rank Math versions, including 1.0.277.x?

We would prefer to verify the migrated data before permanently removing the Rank Math metadata.

Thanks!

**Comment by sybrew** [comment]
Hi @kekkster,

Yes. Transport 1.1.2 is compatible with current Rank Math, including 1.0.277.x, for the post and term fields it migrates (titles, descriptions, Open Graph, Twitter, canonicals, robots, and primary terms). Rank Math has not changed those meta keys or formats.

Optional cleanup is still planned and not shipped. Transport still copies then deletes the Rank Math post/term metadata it touches, so you cannot keep that Rank Math data around for a later verification pass.

I have not removed the beta tag from the extension due to this issue.

We always recommend making a backup before migrations or large data writes, even if no data is lost. 
Also, please ensure that the backup actually works.

**Comment by kekkster** [comment]
Thank you @sybrew for your fast response! 👍 
Just one more question. It says it (Transport) was last updated 3 years ago and compatible with WordPress up to 6.9.
I´m sure you´ve done a ton of work in that time. Just making sure it´s safe to install and will work with 7.1 before we do the switch. :)

<img width="349" height="306" alt="Image" src="https://github.com/user-attachments/assets/a65c4056-aa3a-4ed7-a1ee-e06330938bcc" />

**Comment by sybrew** [comment]
Yes, it still works with 7.1. Nothing in 7.0 or 7.1 affects the plugin, aside from a few visual glitches.
